clang/include/clang/CodeGen/CGFunctionInfo.h

```diff
@@ -608,6 +608,9 @@ class CGFunctionInfo final
   /// Log 2 of the maximum vector width.
   unsigned MaxVectorWidth : 4;
 
+  /// Log2 of ABI_VLEN used in RISCV VLS calling convention.
+  unsigned Log2RISCVABIVLen : 4;
```

Comment:
Don't you need 5 bits for 65536 which is 2^16. 16 is 0b10000 which is 5 bits.

Reply:
Yes, we need 5 bits, thanks for finding out that!